src/main/java/io/anserini/collection/CoreCollection.java

```diff
@@ -144,8 +141,15 @@ public boolean indexable() {
       return true;
     }
 
-    public Map<String, JsonNode> jsonFields() {
-      return jsonFields;
+    public JsonNode jsonNode() {
+      return jsonNode;
+    }
+
+    public String getJsonString(JsonNode json, String key) {
+      if (!json.has(key) || json.get(key).asText() == "null") {
+        return "";
```

Comment:
Is returning `null` better than empty string here?

Reply:
Left it as empty string since I'd rather not index "null"